pkg/securitycontext/accessors.go

```diff
@@ -205,6 +221,7 @@ type ContainerSecurityContextMutator interface {
 	SetPrivileged(*bool)
 	SetSELinuxOptions(*api.SELinuxOptions)
 	SetRunAsUser(*int64)
+	SetRunAsGroup(*int64)
```

Comment:
test in TestContainerSecurityContextMutator and TestEffectiveContainerSecurityContextMutator